pymbar/mbar.py

```diff
@@ -527,12 +548,20 @@ def getFreeEnergyDifferences(self, compute_uncertainty=True, uncertainty_method=
 
         result_vals['Delta_f'] = Deltaf_ij
 
-        if compute_uncertainty or return_theta:
+        if compute_uncertainty and uncertainty_method == 'bootstrap':
+            diffm = np.zeros([self.K,self.K,self.nbootstraps])
+            # take the matrix of differences of f_ij
+            for b in range(self.nbootstraps):
+                diffm[:,:,b] = np.matrix(self.f_k_boots[b,:]) - np.matrix(self.f_k_boots[b,:]).transpose() 
```

Comment:
@mrshirts : Could you clarify why we need to subtract each bootstrapped free energy matrix from its transpose?